api/api.go

```diff
@@ -38,7 +43,11 @@ func CreateAndInitialise(bindAddr, elasticSearchAPIURL string, errorChan chan er
 		return errors.Wrap(errTimeseries, "Failed to setup timeseries templates")
 	}
 
-	api := NewSearchQueryAPI(router)
+	if elasticSearchClient == nil {
```

Comment:
Sorry Im being super nit picky here but I would do this check first - why set up search if the client is nil/invalid. (sorry)

Reply:
Nope, that's totally valid. I actually thought this but then went with '"but it's closer to where I'm using it".